app/scripts/controllers/home.ctrl.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular.module('yourStlCourts').controller('HomeCtrl', function (TicketFinder, $state, municipalities, courts) {
+  angular.module('yourStlCourts').controller('HomeCtrl', function (TicketFinder, $state, municipalities, courts) {
```

Comment:
Not a big deal but this indent shouldn't have been added. And this file didn't change otherwise